packages/common/src/system/guid.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// @ts-ignore
+import * as uuid from 'uuid-with-v6';
+
+// tslint:disable: no-unsafe-any
+
+export namespace Guid {
```

Comment:
you could make these class & inject it via dep injection & mock them for uts

Reply:
too complex to support injection and class instance for the current purposes; u can mock func simple if needed